go/vt/schemadiff/schema_test.go

```diff
@@ -162,7 +162,7 @@ func TestNewSchemaFromQueriesLoop(t *testing.T) {
 	)
 	_, err := NewSchemaFromQueries(queries)
 	require.Error(t, err)
-	err = vterrors.UnwrapFirst(err)
+	err = errors.Unwrap(err)
```

Comment:
I think this change might be causing the issue in the unit test CI:
```
=== RUN   TestNewSchemaFromQueriesLoop
    schema_test.go:166: 
        	Error Trace:	/home/runner/work/vitess/vitess/go/vt/schemadiff/schema_test.go:166
        	Error:      	An error is expected but got nil.
        	Test:       	TestNewSchemaFromQueriesLoop
--- FAIL: TestNewSchemaFromQueriesLoop (0.00s)
```

Reply:
Whoops, didn't intend to commit this change from some testing I did.

Reply:
Got it!